libs/@local/graph/api/src/rest/permissions.rs

```diff
@@ -0,0 +1,237 @@
+//! Web routes for CRUD operations on permissions.
+
+use alloc::sync::Arc;
+
+use axum::{
+    Extension, Router,
+    extract::Path,
+    response::Response,
+    routing::{get, post},
+};
+use hash_graph_authorization::{
+    AuthorizationApiPool,
+    policies::{
+        Policy, PolicyId,
+        store::{PolicyFilter, PolicyStore},
+    },
+};
+use hash_graph_store::pool::StorePool;
+use hash_temporal_client::TemporalClient;
+use http::StatusCode;
+use type_system::principal::actor::ActorId;
+use utoipa::OpenApi;
+
+use crate::rest::{AuthenticatedUserHeader, json::Json, status::report_to_response};
+
+#[derive(OpenApi)]
+#[openapi(
+    paths(
+        get_policy_by_id,
+        query_policies,
+        resolve_policies_for_actor,
+
+        seed_system_policies,
+    ),
+    tags(
+        (name = "Permission", description = "Permission management API")
+    )
+)]
+pub(crate) struct PermissionResource;
+
+impl PermissionResource {
+    /// Create routes for interacting with actors.
+    pub(crate) fn routes<S, A>() -> Router
+    where
+        S: StorePool + Send + Sync + 'static,
+        A: AuthorizationApiPool + Send + Sync + 'static,
+        for<'p, 'a> S::Store<'p, A::Api<'a>>: PolicyStore,
+    {
+        // TODO: The URL format here is preliminary and will have to change.
```

Comment:
nb: should this have a linear issue so that we don't forget?

Reply:
It's unlikely that we would do that before HaRPC lands, so I wouldn't worry about it.